audit_manager.tf

```diff
@@ -0,0 +1,41 @@
+resource "aws_auditmanager_account_registration" "default" {
+  count = var.aws_auditmanager_config.enabled == true ? 1 : 0
+
+  delegated_admin_account = data.aws_caller_identity.audit.account_id
+  deregister_on_destroy   = true
```

Comment:
Shall we add this to the var `aws_auditmanager_config` with default on false? If this is used for audits then you want to make a conscious decision to remove this by setting this to true thereby preventing removal by accident by e.g. refactoring some code.

Reply:
After reading more about this I don't think it would anything. If we manage this with TF we want to deregister on resource destruction. This doesn't affect the assessments and evidence however.

> When you remove a delegated administrator from your Audit Manager settings, you continue to have access to the evidence that you previously collected under that account. This is also the case when you deregister a delegated administrator from AWS Organizations.

Source: https://docs.aws.amazon.com/audit-manager/latest/APIReference/API_DeregisterOrganizationAdminAccount.html